src/components/filter-input/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { ChangeEvent, useRef } from 'react'
+import { IconFilter16 } from '@hashicorp/flight-icons/svg-react/filter-16'
+import { IconX16 } from '@hashicorp/flight-icons/svg-react/x-16'
+import s from './filter-input.module.css'
+
+interface FilterInputProps {
+	placeholder?: string
+	value: string
+	onChange: (newValue: string) => void
+}
+
+const FilterInput: React.FC<FilterInputProps> = ({
```

Comment:
This was copy-pasta'd from `SidebarFilterInput`, with the addition of the `placeholder` prop.